accounts/manager.go

```diff
@@ -33,14 +37,22 @@ type Config struct {
 	InsecureUnlockAllowed bool // Whether account unlocking in insecure environment is allowed
 }
 
+// NewBackendEvent lets the manager know it should
+// track the given backend for wallet updates.
+type NewBackendEvent struct {
+	backend Backend
+	wg      *sync.WaitGroup
+}
```

Comment:
This type doesn't need to be exported. Also, it is not common to use WaitGroup for response signaling, it might be better to use a channel.

Reply:
Good catch re export. Also replaced the wg with a channel.